use-cases/NVIDIA-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,290 @@
+# Using Nvidia GPU device with Kata Containers
```

Comment:
The name of the file you have created makes perfect sense. But, there is an existing `./use-cases/GPU-passthrough-and-Kata.md` for Intel GPUs. As such, I suggest that you create a second commit that does the following:

- Renames `./use-cases/GPU-passthrough-and-Kata.md` to `./use-cases/Intel-GPU-passthrough-and-Kata.md.
- Creates `./use-cases/GPU-passthrough-and-Kata.md` which contains only a few lines of introductory text.

Something like:


----

```md
# Using GPUs with Kata Containers

Kata Containers supports passing certain GPUs from the host into the container. Select the GPU vendor for detailed information:

- [Intel](Intel-GPU-passthrough-and-Kata.md)
- [Nvidia](use-cases/NVIDIA-GPU-passthrough-and-Kata.md)
```

That way, we don't break the existing link but do allow the user to select the most appropriate document.

Reply:
OK